packages/core/src/odata/v2/request-builder/function-import-request-builder.ts

```diff
@@ -13,7 +19,9 @@ export class FunctionImportRequestBuilder<
   // reuse
   ParametersT,
   ReturnT
-> extends ActionFunctionImportRequestBuilderBase<ParametersT, ReturnT> {
+> extends MethodRequestBuilderBase<
```

Comment:
I think here you should not overwrite the changes for the introduction of actions?

Reply:
See also removed files below

Reply:
I have no idea, what happened here, I hope I restored order, but please check again, carefully.